### DISCUSSION TITLE
Adding sensors to app.

### DISCUSSION THREAD

**Discussion by ironscull** [ChilliBits/particulate-matter-app#168] [Support]
Hi. I got this problem with adding my sensors. In configuration I selected correct sending server. App can see my sensors but when i want to add them a message pops up. "This app can import data from sensor if in config You turn on send to Feinstaub App function." I tried few times but no luck i cant add my sensors. I could use some help. Thanks A lot.

**Comment by marcauberer** [comment]
Hi, could you tell me the chipId of your sensor and the version of the installed firmware?

**Comment by ironscull** [comment]
Sensors are connected  to NodeMcu v2. But the  firmware? IDK I installed latest version I think.

**Comment by marcauberer** [comment]
Okay, the latest version would be NRZ-2020-131, but I also need the chip id of your NodeMCU (5-10 digit number, e.g. 4017638). You can find it on the configuration page of your NodeMCU.
![Screenshot_20201119-211332](https://user-images.githubusercontent.com/59527509/99719346-788a4080-2aac-11eb-8f22-90e9b9e3d8f2.png)

**Comment by ironscull** [comment]
This is it?

czw., 19 lis 2020, 21:02 użytkownik Marc Auberer <notifications@github.com>
napisał:

**Reply by marcauberer** [reply]
No! That is not your NodeMCU. I need the id of yours!

**Comment by ironscull** [comment]
Does this could do?

czw., 19 lis 2020, 21:18 użytkownik Marc Auberer <notifications@github.com>
napisał:
